lib/try.js

```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var request = require('request');
+
+module.exports = function() {
+  var proxy = new express.Router();
+  proxy.use(cors());
+  proxy.use(bodyParser());
+  
+  proxy.all('/', function(req, res) {  
+    var url = req.query.url;
```

Comment:
as @lfryc pointed out we should deffo look at making this a header rather than query string param